proposals/core.md

```diff
@@ -116,40 +116,36 @@ To use the new bidirectional protocol, the client must first establish a WebSock
 
 >[RESOLUTION](https://www.w3.org/2019/09/20-webdriver-minutes.html#resolution01): Bi-di is always enabled. An optional capability, defaulting to true, indicating that bi-di is desired. When a new session is established, the return value of the new session contains the new top-level property of the bi-directional URL
 
-A bidirectional session starts as a traditional session created with a POST /session command. The client can upgrade their session to a bidirectional one by connecting to the WebSocket URL provided in the response. The client can also request the optional "protocolVersion" capability to explicitly enable or disable bidi:
+A bidirectional session starts as a traditional session created with a POST /session command. The client can request a bidirectional using the "webSocketUrl" capability:

```

Comment:
Nit: "The client can request a bidirectional session using the "webSocketUrl" capability:"

Reply:
Ah, missing "session".